ConsoleUI/ModListScreen.cs

```diff
@@ -303,6 +303,15 @@ public ModListScreen(GameInstanceManager mgr, bool dbg)
             mainMenu = new ConsolePopupMenu(opts);
         }
 
+        public override void Run(ConsoleTheme theme, Action<ConsoleTheme> process = null)
+        {
+            if (!((Registry) registry)?.HasAnyAvailable() ?? false)
```

Comment:
```suggestion
            if (!registry.HasAnyAvailable())
```

Reading this line makes me think that `registry` has the wrong type. There isn't much benefit from using `IRegistryQuerier`, especially since `Registry` inherits from it. And the object that we retrieve initially is a `Registry` (line 24). Trying it out locally, I think we can change line 616 to store a `Registry` and everything will still work. Does it look OK to you?

Reply:
The `IRegistryQuerier` is kinda worthless right now anyway. I'm currently trying to implement a new "distributed" registry, which splits the local instance data like installed mods and the (theoretically) global data like available modules into different files, to hopefully reduce the chance for corruptions, and an interface only for read methods is pretty much useless since we need to cast it to `Registry` pretty much everywhere. An `IRegistry` would be much easier to handle.

But that's another topic, for here and now just storing it as `Registry` sounds reasonable :+1:

Reply:
> splits the local instance data like installed mods and the (theoretically) global data like available modules into different files

Nice, that has been on my to-do list for years now!